tools/clang/unittests/HLSLExec/ExecutionTest.cpp

```diff
@@ -6954,29 +6923,6 @@ static bool VerifyOutputWithExpectedValueHalf(uint16_t output, uint16_t ref,
       CompareOutputWithExpectedValueHalf(output, ref, type, tolerance));
 }
 
-template <typename T>
-static bool CompareOutputWithExpectedValue(T output, T ref,
```

Comment:
Can we just make these non-static?  This was infrastructure used by WaveMatrix tests that were removed, but I expect that it would be useful again in upcoming tests.

Reply:
We can always add it back when needed.